src/index.ts

```diff
@@ -156,6 +159,9 @@ export const hostPortsVar = (
 	}
 	return hostPairs.split(',').map((hostPair): HostPort => {
 		const [host, maybePort] = hostPair.trim().split(':');
+		if (maybePort == null || host == null) {
+			throw new Error(`Invalid port for '${varName}': ${maybePort}`);
```

Comment:
This message does not make sense for what is actually being checked - it's not just the port that might be missing, and specifically it's missing rather than invalid